sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/EditContactPersonSteps.java

```diff
@@ -0,0 +1,325 @@
+          softly.assertThat(aPerson.getSex()).isEqualTo(CreateNewContactSteps.contact.getSex());
+          softly
+              .assertThat(aPerson.getNationalHealthId())
+              .isEqualTo(CreateNewContactSteps.contact.getNationalHealthId());
+          softly
+              .assertThat(aPerson.getPassportNumber())
+              .isEqualTo(CreateNewContactSteps.contact.getPassportNumber());
+          softly
+              .assertThat(aPerson.getEmailAddress())
+              .isEqualTo(CreateNewContactSteps.contact.getPrimaryEmailAddress());
+          softly
+              .assertThat(aPerson.getPhoneNumber())
+              .isEqualTo(CreateNewContactSteps.contact.getPrimaryPhoneNumber());
+          softly.assertAll();
+        });
+
+    Then(
+        "I complete all default empty fields from Contact Person tab",
+        () -> {
+          newGeneratedPerson = personService.buildGeneratedPerson();
+          fillSalutation(newGeneratedPerson.getSalutation());
+          fillDateOfBirth(newGeneratedPerson.getDateOfBirth());
+          selectSex(newGeneratedPerson.getSex());
+          selectPresentConditionOfPerson(newGeneratedPerson.getPresentConditionOfPerson());
+          fillPassportNumber(newGeneratedPerson.getPassportNumber());
+          fillNationalHealthId(newGeneratedPerson.getNationalHealthId());
+          fillExternalId(newGeneratedPerson.getExternalId());
+          fillExternalToken(newGeneratedPerson.getExternalToken());
+          fillExternalToken(newGeneratedPerson.getExternalToken());
+          selectTypeOfOccupation(newGeneratedPerson.getTypeOfOccupation());
+          selectStaffOfArmedForces(newGeneratedPerson.getStaffOfArmedForces());
+          selectEducation(newGeneratedPerson.getEducation());
+          selectRegion(newGeneratedPerson.getRegion());
+          selectDistrict(newGeneratedPerson.getDistrict());
+          selectCommunity(newGeneratedPerson.getCommunity());
+          selectFacilityCategory(newGeneratedPerson.getFacilityCategory());
+          selectFacilityType(newGeneratedPerson.getFacilityType());
+          selectFacility(newGeneratedPerson.getFacility());
+          fillFacilityNameAndDescription(newGeneratedPerson.getFacilityNameAndDescription());
+          fillStreet(newGeneratedPerson.getStreet());
+          fillHouseNumber(newGeneratedPerson.getHouseNumber());
+          fillAdditionalInformation(newGeneratedPerson.getAdditionalInformation());
+          fillPostalCode(newGeneratedPerson.getPostalCode());
+          fillCity(newGeneratedPerson.getCity());
+          selectAreaType(newGeneratedPerson.getAreaType());
+          fillContactPersonFirstName(newGeneratedPerson.getContactPersonFirstName());
+          fillContactPersonLastName(newGeneratedPerson.getContactPersonLastName());
+          fillContactPersonPhoneNumber(newGeneratedPerson.getContactPersonPhoneNumber());
+          fillContactPersonEmailAddress(newGeneratedPerson.getContactPersonEmailAddress());
```

Comment:
Because these two fields does not have css selector for this faze of initiative we will not check.(phone and email).
That means we will not fill/check them.

Reply:
removed